### ISSUE TITLE
Relative / base paths option

### ISSUE THREAD

**Issue by andy-pi** [homeisfar/LinuxDir2HTML#9]
Thanks for making this. I found it whilst searching for a linux replacement for my windows dir2html. My use case is to use as an index for files in an s3 bucket, but I generate the file locally. Windows dir2html had an option to create relative paths, so that the html links would work wherever they were ultimately placed on the filesystem or internet. I tried just a find and replace on the output html but didn't work for some reason, and I couldn't see an easy way to change it in the source code. Is this something you might implement?

**Comment by homeisfar** [comment]
Are you saying something like this:

Your `pwd` is `/home/localcomputer/BunchOfFiles/` and you create the html output into this directory. So the expected file links are `./file1` `./file2` instead of `/home/localcomputer/BunchOfFiles/file1`.

Therefore, you can now transfer BunchOfFiles/ to another destination and reuse the links. Am I understanding correctly?

**Comment by andy-pi** [comment]
yeah exactly. I want to generate something on my local filesystem and then upload it to s3. Would be amazing if this is possible!

**Comment by homeisfar** [comment]
Didn't mean to leave this untouched for so long. If you still have interest two years later, let me know.

**Comment by chrootlogin** [comment]
It would be very nice if you could do it, because I use it to list downloadable files

**Comment by chrootlogin** [comment]
I did a PR which should fix it https://github.com/homeisfar/LinuxDir2HTML/pull/19 you can use --protocol https://my-domain.com/subdir
